README.md

```diff
@@ -22,3 +28,17 @@ version = 375
  status = new
   names = fault-tolerant-execution-task-memory-growth-factor, optimizer.non-estimatable-predicate-approximation.enabled
 ```
+
+### Update the property index 
+
+In case that the local properties index of the project does not include the properties corresponding to the latest Trino release,
+run `properties/update.sh` to retrieve and save locally those properties.
+This operation requires retrieving several versions of the `io.trino:trino-server:${version}:tar.gz` artifacts to the local
+Maven repository of the machine on which the tool is being used and may require multiple minutes to complete.
+
+Run `properties/update.sh -s 466 -t 468` to upsert locally in the properties local index the properties corresponding to the Trino 
```

Comment:
Avoid upsert, it's a commonly used word, we don't use SQL to write the data, and it doesn't update properties, just overwrites them.

Reply:
BTW there's also a workflow to do this on GitHub runners, to utilize its fast internet connection, and ephemeral storage.